routes.py

```diff
@@ -20,6 +20,8 @@
 
 def check_call_sign(key, value):
     """Raise exception if value is valid pubkey and can not be used as call sign."""
+    if webserver.validation.DEBUG:
+        return value
```

Comment:
@yaroslavr-itd , Don't you think it is cleaner to simple not set the checker on debug mode?
```python
if not DEBUG:
    webserver.validation.KWARGS_CHECKERS_AND_FIXERS['call_sign'] = check_call_sign
```

Reply:
This version is better because it is checked in runtime, and not on import time.